core/src/main/java/no/nordicsemi/android/kotlin/ble/core/wrapper/NativeBluetoothGattDescriptor.kt

```diff
@@ -60,9 +61,9 @@ data class NativeBluetoothGattDescriptor(
     override fun toString(): String {
         return StringBuilder()
             .append("{")
-            .append("uuid: $uuid")
-            .append("permissions: $permissions")
-            .append("value: ${value.toDisplayString()}")
+            .append("uuid : $uuid ")
```

Comment:
why no spaces here after initial "?

Reply:
Permissions object has spaces has already, you may check on logcat.